ontologies/works.ttl

```diff
@@ -18,48 +18,52 @@ TBC
-	rdf:isDefinedBy <TBC/works> .
-	
-ww:Collection rdf:type owl:Class ;
-	rdfs:label "Collection"@en ;
-	rdfs:comment "A grouping of material comprising more than one work, the grouping being either organic (i.e., a Fonds) or artificial."@en ;
-	rdfs:subClassOf ww:Work
-	rdf:isDefinedBy <TBC/works> .
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
 	
 ww:Item rdf:type owl:Class ;
 	rdfs:label "Item"@en ;
 	rdfs:comment "A specific instance of a work: for instance, an individual physical copy with its own location.  This corresponds to the bottom, Instance, layer of FRBR.  Works that are unique - that is, have only one copy - are described in terms of a Work for their intellectual content, linked to an Item describing their physical characteristics."@en ;
-	rdf:isDefinedBy <TBC/works> .	
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .	
 	
-ww:Agent rdf:type owl:Class ;
-	rdfs:label "Agent"@en ;
-	rdfs:comment "An individual or organisation that stands in some relationship to a work."@en ;
-	rdf:isDefinedBy <TBC/works> .
+ww:Identifier rdf:type owl:Class ;
+	rdfs:label "Identifier"@en ;
+	rdfs:comment "A unique system-generated identifier for entities as described within this ontology, that will be used to ensure that appropriate pieces of data in different systems can be related to each other."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .		
 
+ww:WorkType rdf:type owl:Class ;
+	rdfs:label "WorkType"@en ;
+	rdfs:comment "A broad, top-level description of the form of a work: namely, whether it is a printed book, archive, painting, photograph, moving image, etc."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
+	
 ww:Format rdf:type owl:Class ;
 	rdfs:label "Format"@en ;
-	rdfs:comment "The format of a work: namely, whether it is a printed book, archive, image, moving image, etc."@en ;
-	rdf:isDefinedBy <TBC/works> .
+	rdfs:comment "Format is used to provide a detailed expansion of a WorkType, in terms of physical characteristics, rendering requirements or both: thus, WorkType would state that a work comprises moving image material, whilst Format explains that it is a VHS video."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
 	
-ww:Location rdf:type owl:Class ;
-	rdfs:label "Location"@en ;
-	rdfs:comment "The place at which a work was published or otherwise created."@en ;
-	rdf:isDefinedBy <TBC/works> .
+ww:Place rdf:type owl:Class ;
+	rdfs:label "Place"@en ;
+	rdfs:comment "A physical location that relates in some way to a work: for example, the place at which a work was published or otherwise created."@en ;
```

Comment:
Alas no: for the moment, like the link to Person authority, CALM just allows you to say that this place/person has "something to do" with the record.  This as you'll recall is something I've fed into the Requests for Enhancement and will be banging on about again at the Axiell conference in a couple of weeks!  For CALM, this sort of information will generally just be buried in the free-text description and this particular property won't be needed: it'll come into its own when we start looking at Sierra data, in which the MARC 260 field for publication details splits up the place, publisher and year in nice predictable ways.